styles.css

```diff
@@ -891,12 +900,15 @@ textarea {
 
   .popup2 .link-img img {
     height: 40px;
+    padding-left: 60px;
+    margin-top: 180px;
+    /* height: 40px;
     position: absolute;
-    bottom: 80px;
+    bottom: 80px; */
```

Comment:
Great job implementing some of the requested changes! As requested by the previous reviewer, please make sure that your buttons are visible on all mobile devices 😉. Currently, there is an overlapping happening in some widths:
![image](https://user-images.githubusercontent.com/42799579/211104741-fa2dad02-da4a-4c83-bef7-301824562d7a.png)